data/pages/faq-grantees.mdx

```diff
@@ -105,11 +105,11 @@ evaluate progress.
 
 ### What does an ideal progress report look like?
 
-An ideal progress report is:
-
 - Written in Markdown format
 - Between 1 and 3 pages
-- Enriched with links to pull requests, commits, and other relevant work produced.
+- Enriched with the most relevant links to pull requests, commits, and other work produced.
+- Has a brief summary in each section before diving into links in order to help us contextualize the links provided.
```

Comment:
- Has a brief summary in each section before providing additional detail (links, PRs, etc)

Reply:
[Commit here](https://github.com/OpenSats/website/pull/335/commits/ee29b8e56c46b82699468bfafb36855192f4aecb)